src/DetailsView/reports/components/report-sections/outcome-summary-bar.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { kebabCase } from 'lodash';
+import { CheckIcon } from '../../../../common/icons/check-icon';
+import { CircleIcon } from '../../../../common/icons/circle-icon';
+import { CrossIcon } from '../../../../common/icons/cross-icon';
+import { NamedSFC } from '../../../../common/react/named-sfc';
+import { SectionProps } from './report-section-factory';
+
+const iconMap = {
+    pass: <CheckIcon />,
+    'not applicable': <CircleIcon />,
```

Comment:
Is there any reason that only ``not applicable`` is inside quotes?

Reply:
yeap... the space